app/component/map/tile-layer/ChargingStations.js

```diff
@@ -61,7 +61,15 @@ class ChargingStations {
 
   drawStatus = ({ geom, properties }) => {
     if (this.tile.coords.z <= this.config.chargingStations.smallIconZoom) {
-      return drawStopIcon(this.tile, geom, 'charging-station');
+      return drawStopIcon(
+        this.tile,
+        geom,
+        'charging-station',
+        null,
+        null,
+        null,
+        { 'mode-charging-station': '#00b096' },
```

Comment:
Could you define color in `'config.hbnext.js` line 92 instead of adding it here?

Reply:
That would be a good idea, but we should do that for all the icons. I could start with this one.

Reply:
I already did that for bus, car and rail:

```
colors: {
        primary: '#9fc727',
        iconColors: {
            'mode-bus': '#ff0000',
            'mode-car': '#007AC9',
            'mode-rail': '#008000',
        },
    },
```

Reply:
I will add the rest of the colors, then.